src/sidebar/panel.js

```diff
@@ -201,14 +233,68 @@ function getThemeFromStorage() {
     }
   });
 }
+
+function getLastSyncedTime() {
+  const getting = browser.storage.local.get(['last_modified', 'credentials']);
+  getting.then(data => {
+    if (data.hasOwnProperty('credentials')) {
+      const time = new Date(data.last_modified).toLocaleTimeString();
+      enableSync.textContent = 'Synced at ' + time;
+    }
+  });
+}
+
 document.addEventListener('DOMContentLoaded', getThemeFromStorage);
+document.addEventListener('DOMContentLoaded', getLastSyncedTime);
```

Comment:
Should we have two listeners for the event, or just a single listener which calls the two functions?

For example:

```js
document.addEventListener('DOMContentLoaded', () => {
  getThemeFromStorage(),
  getLastSyncedTime()
});
```

Reply:
I like to have two so that it is easiser to remove one eventually.